qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/helper/DocumentIntents.java

```diff
@@ -19,31 +19,32 @@
  */
 public class DocumentIntents {
 
-	public void handleSaveFileIntent(File file) {
+    public void handleSaveFileIntent(File file) {
 
-		Intent data = new Intent();
-		data.setData(Uri.fromFile(file));
+        Intent data = new Intent();
+        data.setData(Uri.fromFile(file));
 
-		Intents.intending(allOf(
-				hasAction(Intent.ACTION_CREATE_DOCUMENT),
-				hasCategories(hasItem(Intent.CATEGORY_OPENABLE))
-		)).respondWith(
-				new Instrumentation.ActivityResult(Activity.RESULT_OK, data)
-		);
+        Intents.intending(allOf(
```

Comment:
pls use tabs instead of 4 spaces... for all commits. pls configurate your android studio